js/api.js

```diff
@@ -0,0 +1,47 @@
+export const getData = (onSuccess, onError) => {
+  fetch('https://25.javascript.pages.academy/keksobooking/data',
+
+    {
+      method: 'GET',
+      credentials: 'same-origin',
+    },
+
+  )
+    .then((response) => response.json())
+
+    .then((adds) =>  {
```

Comment:
```suggestion
    .then((data) =>  {
```

вся твоя ф-я называется - получение данных - и дальше ты взаимодействуешь с данными, тебе пофигу, что это рекламные объявления, ф-я она универсальная и данные могут придти любые